### PR TITLE
AV::Base use #empty

### PR DESCRIPTION
**Pull request by ahorek**
reference: https://github.com/rails/rails/commit/e17fe52e0e0ef0842b6c409e1110a862c4e005bc#diff-0f66ef96762e67b6746a49de6182c1a4

fixes
```
  1) RSpec::Rails::HelperExampleGroup#helper includes ApplicationHelper
     Failure/Error: example.run

     RuntimeError:
       Warnings were generated: DEPRECATION WARNING: ActionView::Base instances should be constructed with a lookup context,
       assignments, and a controller.
```

### CODE REVIEW

File: spec/rspec/rails/example/helper_example_group_spec.rb [hunk 1]
```diff
@@ -42,7 +42,11 @@ class InternalClass
         group = RSpec::Core::ExampleGroup.describe do
           include HelperExampleGroup
           def _view
-            ActionView::Base.new
+            if ActionView::Base.respond_to?(:empty)
```
Comment (JonRowe) [inline spec/rspec/rails/example/helper_example_group_spec.rb]: @benoittgt we typically define methods conditionally for this sort of thing, fancy refactoring it in the dev branch?

Comment (benoittgt) [reply spec/rspec/rails/example/helper_example_group_spec.rb]: I am sorry @JonRowe I didn't see your message. What do you mean by "define methods conditionally" in that case?

Thanks

Comment (JonRowe) [reply spec/rspec/rails/example/helper_example_group_spec.rb]:
```Ruby
if ActionView::Base.respond_to?(:empty)
  def _view
    ActionView::Base.empty
  end
else
  def _view
    ActionView::Base.new
  end
end
```

Comment (benoittgt) [reply spec/rspec/rails/example/helper_example_group_spec.rb]: Ohhh ok. Thanks for your patience. 

I made the change in https://github.com/rspec/rspec-rails/pull/2071/commits/bf65b86556c2bdd152962ac5ad17c4dcbb74b534

**Pull request merged**